main.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
Yeah i'm not fan of main.go in the top of the repo

move it to the cmd folder and employ filenames similar to https://github.com/openshift/builder/tree/master/cmd

Reply:
updated in current push, ptal